modules/react-native-status/desktop/rctstatus.cpp

```diff
@@ -490,3 +490,17 @@ void RCTStatus::chaosModeUpdate(bool on, double callbackId) {
             d->bridge->invokePromiseCallback(callbackId, QVariantList{result});
         }, on, callbackId);
 }
+
+QString RCTStatus::generateAlias(QString publicKey) {
+    Q_D(RCTStatus);
+    qCDebug(RCTSTATUS) << "::generateAlias call";
+    //return GenerateGfycat(publicKey.toUtf8().data());
+    return "test";
```

Comment:
@cammellos, was this intentionally left commented?

Reply:
@vkjr Yes, I could not run desktop locally, I was told that is currently not working and adviced to leave it as it is, so I can't really tests this methods, (they need to be synchronous, not async, not sure how desktop handles this), I can take a look at it, but I might need some help from you getting stuff going

Reply:
Ok, I'll take care of them :)